ddns-start

```diff
@@ -3,11 +3,12 @@
 APIKEY="apikey"
 DOMAIN="domain"
 HOST="host"
+FULLDOMAIN=$HOST.$DOMAIN
```

Comment:
If $HOST is blank, will $FULLDOMAIN start with an unnecessary . like ".example.com"?

Reply:
Fuuck. Yeah, it will. Gotta refine this.

Reply:
I literally made one version which worked only with plain domains and another one which was a copy of the original. Whatever, now it should be set.